wiki/Article_Styling_Criteria/en.md

```diff
@@ -299,33 +460,34 @@ You can use the URL from the address bar of your browser and, optionally, omit t
 
 ##### User profiles
 
-All usernames must be linked on first occurrence. Other occurrences are optional, but must be consistent. However, if it is impossible or is hard to determine the user id (because they had changed their username and/or were unlisted from a group), you may skip over it.
+All usernames must be linked on first occurrence. Other occurrences are optional, but must be consistent throughout the entire article for all usernames. However, if it is impossible or hard to determine the user id (because they had changed their username and/or were unlisted from a group), you may skip over it.
```

Comment:
> However, if it is impossible or hard to determine the user id (because they had changed their username and/or were unlisted from a group), you may skip over it.

This conflicts with the next paragraph.